portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,40 @@
 <!DOCTYPE html>
-<html>
-  <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
-  </body>
-</html>
+<meta charset="utf-8">
+<title>My Portfolio</title>
+<link rel="stylesheet" href="style.css">
+<div id="header-content">
+  <div id="image-container">
+    <img src="images/main-selfie.jpg" alt="Picture of me" id="header-selfie">
+    <button id="cycle-button" onclick="cyclePictures()"> Cycle Pictures </button>
+  </div>	
+  <div id="greeting-container">
+    <h1>Welcome to my portfolio</h1>
+    <h3>My name is Malachi Brewer. I am a sophomore at the University of Georgia.</h3>
+    <h3>Welcome Buds! I created this site to house random pieces of information about me for anyone who might be curious. 
+            Clicking on the buttons below will take you to different parts of the page.</h3> 
+    <ul id="header-nav"> 
+      <li><a href="#about-me">About Me</a>
+      <li><a href="#education">Education</a>
+      <li><a href="#professional">Professional</a>
+      <li><a href="#random-facts">Random Facts</a>
+    </ul>
+  </div>
+  </div>
+    <h1 id="about-me"> About Me </h1> 
```

Comment:
Something is not right here. If `</div>` is closing `#header-content` then the indentation needs to be fixed here and in the rest of the file following these lines.

Reply:
Done.